src/signalcorr.jl

```diff
@@ -110,18 +106,18 @@ When left unspecified, the lags used are the integers from 0 to
 
 The output is not normalized. See [`autocor`](@ref) for a function with normalization.
 """
-function autocov(x::RealVector, lags::IntegerVector; demean::Bool=true)
+function autocov(x::AbstractVector, lags::AbstractVector{<:Integer}; demean::Bool=true)
     out = Vector{float(eltype(x))}(undef, length(lags))
-    autocov!(out, x, lags; demean=demean)
+    autocov!(out, x, lags; demean)
```

Comment:
This requires Julia >= 1.4


```suggestion
    autocov!(out, x, lags; demean=demean)
```